### PR TITLE
Feature/changing cancelling appointment

### PR DESCRIPTION
**Pull request by shea-maykinmedia**
SDK Pull request for https://github.com/open-formulieren/open-forms/pull/609

**Screenshots**

Screenshot 2021-09-06 at 10 48 43
Screenshot 2021-09-06 at 10 48 49
Screenshot 2021-09-06 at 11 20 44

### CODE REVIEW

File: src/components/CancelAppointment.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+          <p>
+            <FormattedMessage id="uwAfspraakOp" defaultMessage="U staat op het punt om uw afspraak op " />
+            {<FormattedDate
+              value={time}
+              day="numeric"
+              month="long"
+            />}
+            <FormattedMessage id="om" defaultMessage=" om " />
+            {new Date(time).toLocaleTimeString().replace(":", ".").split(":")[0]}
+            <FormattedMessage id="teAnnuleren" defaultMessage="  te annuleren. Vul ter validatie uw e-mailadres in." />
```
Comment (shea-maykinmedia) [inline src/components/CancelAppointment.js:38]: I'm not sure if there is a better way to build up this sentence.  I tried a few different things but couldn't find a better solution.

I can spend more time on it but didn't want to put too much time into this without a second opinion.

File: src/components/CancelAppointment.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+          </p>
+
+          <VerticalSpacer pixels={60} />
```
Comment (shea-maykinmedia) [inline src/components/CancelAppointment.js:41]: Not sure how we feel about a pattern like this?  

It seemed like a simple enough solution to add vertical spacing while also making a reusable solution that can be used in other places.

**Comment by sergei-maertens** [comment]
will review this tomorrow

**Comment by sergei-maertens** [comment]
didn't get to it, next week...

**Comment by sergei-maertens** [comment]
image

Server-side validation errors are now styled in the same way as Formio errors.

**Review by sergei-maertens — APPROVED** [review]

File: src/components/CancelAppointment.js (hunk 1 above)
Comment (sergei-maertens) [reply src/components/CancelAppointment.js]: as discussed in the office - the approach with `values` and `FormattedDate` components etc. into a bigger `FormattedMessage` is the way to go about this.

**Pull request merged**